### ISSUE TITLE
Export on MacOS doesn't keep markdown code image width

### ISSUE THREAD

**Issue by kaosweaver** [KosmosisDire/obsidian-webpage-export#227]
**Describe the bug**
When using the markdown code for image widths, the exported HTML doesn't limit the width of the image to the specified width. The export in the prior version kept the set width for the image as specified. 

**Operating System**
MacOS (latest)

**To Reproduce**
Set markdown code for an image that is wider than the specified width in one page like:
`![Elf Fire Swinger|320](https://dort.world/images/FireSwingerElf.jpg)`
Export the page (I exported the entire site, less a few folders, none of them image related)
View the page, the image will display to its actual size.

**Screenshots**
<img width="798" alt="image" src="https://github.com/KosmosisDire/obsidian-webpage-export/assets/25509704/22cc86a5-125a-43d5-9e19-6d96975e906a">

<img width="806" alt="image" src="https://github.com/KosmosisDire/obsidian-webpage-export/assets/25509704/7912e5bc-627f-44bc-ae05-ab2e9dc1843d">

You can view the export here:
https://dort.world/funta/fire-swingers.html

**Comment by KosmosisDire** [comment]
Hmm interesting, this still works in my tests on windows and this doesn't seem like something that should be OS specific. It may be the format of the link I will test a few things. Also, you have a beautiful vault set up! (BTW, if you do a full re-export with the latest version it should fix the scrollbars so they will look like they do in obsidian)

**Comment by KosmosisDire** [comment]
I am not able to reproduce this, not sure what is wrong but I will look into it in the future

**Comment by kaosweaver** [comment]
Is there anything I can do to provide more insight for you?  I've exported it again.
I have included Fantasy Statblocks as an additional CSS. (I can send shots of the parameters I'm using if you'd like, I could send you the .md files as well - whatever you need, just let me know.

Also, thanks on the compliment on the vault. Most of the export beauty is from your work. Very impressed.

**Comment by KosmosisDire** [comment]
I will try and look into it today and I'll let you know if there is anything you can give me to help :) Thanks!

**Comment by wyne** [comment]
I'm using Mac and I'm observing the same thing. Image sizes aren't being respected.

I'm happy to help investigate if pointed in the right direction.

**Comment by KosmosisDire** [comment]
Sure, let me know if you find anything. You should investigate on the 1.7.4 branch as that is where all the latest development is. Thanks! :)

**Comment by wyne** [comment]
I noticed something suspicious looking at `main.js` in my plugins directory on line 56393:

```js
    html.querySelectorAll("img, video, svg").forEach((element) => {
      let width = element.getAttribute("width");
      if (width) {
        element.style.width = width || "100%";
        element.style.maxWidth = "100%";
        element.removeAttribute("width");
      }
    });
```

This seems backwards compared to this: 
https://github.com/KosmosisDire/obsidian-webpage-export/blob/167389ec4823dde609488b90d83cca7d6e26bc52/scripts/html-generation/markdown-renderer.ts#L414

**Comment by wyne** [comment]
I ran the 1.7.4 branch locally and can confirm that it fixes the issue.

**Comment by KosmosisDire** [comment]
This should be fixed in 1.8.0. Let me know if it still does not work as expected and reopen the issue. If you find any new issues please submit a new issue :)
Thanks!
